lambda-deployment-orca/lambda-deployment-orca.gradle

```diff
@@ -28,19 +28,19 @@ dependencies {
   compileOnly (group: 'com.netflix.spinnaker.orca', name: 'orca-api', version: "${orcaVersion}")
   compileOnly (group: 'com.netflix.spinnaker.orca', name: 'orca-clouddriver', version: "${orcaVersion}")
   compileOnly (group: 'com.netflix.spinnaker.orca', name: 'orca-core', version: "${orcaVersion}")
-  compileOnly ("org.projectlombok:lombok:1.18.12")
-  compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-artifacts', version: "${korkVersion}")
-  compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-plugins-spring-api', version: "${korkVersion}")
-  implementation("com.amazonaws:aws-java-sdk:1.11.877")
```

Comment:
Thanks for upgrading kork.  I still think it's correct to leave these dependencies here, but without their versions.  It'll be interesting to see which ones kork doesn't supply (if any).

Reply:
Done!

Reply:
Sorry, I still see
```
  implementation("org.apache.commons:commons-lang3:3.8.1")
  implementation("com.github.ben-manes.caffeine:guava:2.8.5")
  implementation("com.netflix.awsobjectmapper:awsobjectmapper:1.11.877")
  implementation("org.apache.httpcomponents:httpclient:4.5.9")
  implementation("org.apache.httpcomponents:httpcore:4.4.11")
  implementation("com.squareup.okhttp3:okhttp:4.2.2")
  implementation("com.squareup.okhttp3:okhttp-apache:3.13.1")
```
being deleted, but no replacement of the equivalent dependency without a version.

Reply:
oh your right! i thought the code wasn't using them, after a double check i see only 
```
implementation("com.github.ben-manes.caffeine:guava:2.8.5")
implementation("org.apache.httpcomponents:httpclient:4.5.9")
implementation("org.apache.httpcomponents:httpcore:4.4.11")
```
should be deleted